service/storage/auxiliary_test.go

```diff
@@ -195,18 +195,19 @@ func TestLibrary_Retrieve(t *testing.T) {
 		err := insertKeyValue(t, db, testKey, testValue)
 		require.NoError(t, err)
 
-		l := &Library{
-			codec: &mocks.Codec{
-				UnmarshalFunc: func(b []byte, v interface{}) error {
-					assert.Equal(t, wantEncodedValue, b)
+		codec := mocks.BaselineCodec(t)
+		codec.UnmarshalFunc = func(b []byte, v interface{}) error {
+			assert.Equal(t, wantEncodedValue, b)
 
-					ptr, ok := v.(*uint64)
-					assert.True(t, ok)
+			ptr, ok := v.(*uint64)
+			assert.True(t, ok)
```

Comment:
`assert` doesn't stop the execution here, so we can still crash on the next line. Should use `require` if we don't want to put a conditional clause.